plugins/Files/js/sagas/files.js

```diff
@@ -132,6 +134,7 @@ function* downloadFileSaga(action) {
 		if (action.file.type === 'file') {
 			yield siadCall({
 				url: '/renter/download/' + encodeURI(action.file.siapath),
+				timeout: 60000,
```

Comment:
this should be much longer; something ridiculous like 1 week (or more).

Although, I'm not sure how soon we plan to make the download call non-blocking. If we can get that in before the next release, then we wouldn't need a huge timeout here. Still, it's probably better to change it twice than to risk forgetting about it.